superset-frontend/src/components/Menu/LanguagePicker.tsx

```diff
@@ -46,17 +46,23 @@ export default function LanguagePicker({
         </span>
       }
     >
-      {Object.keys(languages).map(langKey =>
-        langKey === locale ? null : (
-          <MenuItem key={langKey} href={languages[langKey].url}>
-            {' '}
-            <div className="f16">
-              <i className={`flag ${languages[langKey].flag}`} /> -{' '}
-              {languages[langKey].name}
-            </div>
-          </MenuItem>
-        ),
-      )}
+      <Menu
+        onSelect={({ key }) => {
+          window.location.href = languages[key].url;
+        }}
+      >
+        {Object.keys(languages).map(langKey =>
+          langKey === locale ? null : (
+            <Menu.Item key={langKey}>
+              {' '}
+              <div className="f16">
```

Comment:
Any idea what this `f16` class does?

Reply:
`.f16 .flag-code` and `.f16 .flag` provide styles for the flag icons